registration/include/pcl/registration/impl/default_convergence_criteria.hpp

```diff
@@ -120,6 +118,17 @@ pcl::registration::DefaultConvergenceCriteria<Scalar>::hasConverged ()
     }
   }
 
+  if (is_similar)
+  {
+    // Increment the number of transforms that the thresholds are allowed to be similar
+    ++iterations_similar_transforms_;
+  }
+  else
+  {
+    // When the transform becomes large, reset.
+    iterations_similar_transforms_ = 0;
+  }
+
```

Comment:
@SunBlack Should I remove brackets in this code too? I wrapped because there are two sentences with comments, but this is logically one sentence.